### PR TITLE
vitalii.synytskyi

### PR DESCRIPTION
**Pull request by VitaliySynytskyi**

### CODE REVIEW

File: .vs/ProjectSettings.json
Comment (aslobodyanuk) [inline .vs/ProjectSettings.json:1]: Do i need this file in my repo for the project to work ? (you need to remove all unnecessary files)

File: src/services/students/vitalii-synytskyi.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        { x: 11, y: 9, color: Colour.White }, { x: 11, y: 10, color: Colour.White }, { x: 11, y: 11, color: Colour.White }, { x: 11, y: 12, color: Colour.White }, { x: 11, y: 13, color: Colour.White }, { x: 11, y: 14, color: Colour.White }, { x: 11, y: 15, color: Colour.White },
+        { x: 12, y: 3, color: Colour.White }, { x: 12, y: 5, color: Colour.White }, { x: 12, y: 10, color: Colour.White }, { x: 12, y: 12, color: Colour.White },
+        { x: 13, y: 3, color: Colour.White }, { x: 13, y: 5, color: Colour.White }, { x: 13, y: 10, color: Colour.White }, { x: 13, y: 12, color: Colour.White },
+        { x: 14, y: 3, color: Colour.White }, { x: 14, y: 5, color: Colour.White }, { x: 14, y: 10, color: Colour.White }, { x: 14, y: 12, color: Colour.White },
+        { x: 15, y: 4, color: Colour.White }, { x: 15, y: 5, color: Colour.White }, { x: 15, y: 6, color: Colour.White }, { x: 15, y: 7, color: Colour.White }, { x: 15, y: 8, color: Colour.White }, { x: 15, y: 9, color: Colour.White }, { x: 15, y: 10, color: Colour.White }, { x: 15, y: 11, color: Colour.White },
+    ];
+
+
+    for (let frameNumber = 0; frameNumber < Constants.MaxTotalFrames; frameNumber++) {
```
Comment (aslobodyanuk) [inline src/services/students/vitalii-synytskyi.ts:84]: Really, just 3 frames to represent your entire project ? I demand something more interesting.

File: .vs/ProjectSettings.json
Comment (VitaliySynytskyi) [reply .vs/ProjectSettings.json:1]: kk

File: src/services/students/vitalii-synytskyi.ts (hunk 1 above)
Comment (VitaliySynytskyi) [reply src/services/students/vitalii-synytskyi.ts:84]: I can add more frames if needed

**Review by VitaliySynytskyi — COMMENTED** [review]
fixed

File: src/services/students/vitalii-synytskyi.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    public loadConfig(): IAnimationConfig {
+        return {
+            name: 'Vitalii',
+            surname: 'Synytskyi',
+            email: this.getStudentEmail(),
+            projectName: 'Cactus',
+            personalProjectLink: 'https://codeflow.azurewebsites.net',
+            header: Constants.DefaultConfig.header,
```
Comment (aslobodyanuk) [inline src/services/students/vitalii-synytskyi.ts:25]: Do not use default header please.

**Comment by VitaliySynytskyi** [comment]
add header and two new frames

**Pull request merged**